supporting/concentration.py

```diff
@@ -87,7 +87,8 @@ def __init__(self, emis_obj, isrm_obj, detailed_conc_flag, run_parallel, output_
         self.run_calcs = run_calcs
         self.output_dir = output_dir
         self.output_emis_flag = output_emis_flag
-        self.boundary = gpd.read_feather(self.output_geometry_fps[self.output_resolution]).to_crs(self.crs)
+        if self.output_resolution != 'ISRM':
```

Comment:
@amyryao - I was getting an error when I tried to run this at full grid resolution. I'm not sure if you were getting this before? I apologize that this edit is going in a completely unrelated issue.